fetch.js

```diff
@@ -267,13 +267,16 @@
 
   function headers(xhr) {
     var head = new Headers()
-    var pairs = xhr.getAllResponseHeaders().trim().split('\n')
```

Comment:
I agree that this approach would be more elegant than wrapping everything in a `typeof` check.